Assets/_Scripts/Managers/GameManager.cs

```diff
@@ -69,10 +84,48 @@ protected override void Start()
         }
     }
 
+    // When called in canvas manager it changes the scene. 
+    public void ChangeScene(int buildIndex)
+    {
+        SceneManager.LoadScene(buildIndex);

```

Comment:
I think using the explicit name of the scene is a lot clearer and easier to reason about and also means if the build settings change and the order in which the scenes are loaded is changed, this method isn't affected.

[Can use the overload for `LoadScene`](https://docs.unity3d.com/ScriptReference/SceneManagement.SceneManager.LoadScene.html):
```csharp
SceneManager.LoadScene(sceneName);
```